xml/deployment_installation.xml

```diff
@@ -715,26 +741,352 @@
+    <title>Installing from Pre-installed Disk Images</title>
+    <para>
+     For building clusters from virtual machines on supported hypervisors, it
+     is not necessary to individually install each node. &suse; offers
+     pre-installed VM disk images in the following formats: 
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>&kvm; and &xen;</term>
+      <listitem>
+       <para>
+        In QCOW2 format, for &kvm; and for &xen; using full virtualization.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>&xen;</term>
+      <listitem>
+       <para>
+        In QCOW2 format, for &xen; using paravirtualization.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>VMware</term>
+      <listitem>
+       <para>
+        In VMDK format, for VMware ESXi.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>
+       VHD
+      </term>
+      <listitem>
+       <para>For Microsoft Hyper-V.</para>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+    <sect4>
+     <title>Overview</title>
+     <para>
+      When deploying a cluster node from pre-installed disk images, the setup
+      program never runs. Thus, it is not possible to set values such as time
+      servers, node names, the &rootuser; password for the &admin_node; and so
+      on in the normal way.
+     </para>
+     <para>      
+      Instead, these values must be set using <literal>cloud-init</literal> configuration
+      files, which are placed into a specific subdirectory within an ISO image
+      which is attached to the VM as a secondary disk. The pre-installed
+      operating system on the disk image looks for a medium with the
+      volume label <literal>cidata</literal>, searches that volume for the
+      configuration files, and applies the settings within them.
+     </para>
+     <para>
+      Full details of the cloud-init metadata files can be found in the
+      Configuration chapter of this manual; for more information, see
+      <xref linkend="cloud-init.configuration"/>.
+     </para>
+     <para>
+      As with individual node installation, the &admin_node; must be configured
+      separately and differently from the &worker_node;s.
+     </para>
+    </sect4>
+    <sect4>
+     <title>Procedure</title>
+     <para>
+      The process is very similar for all hypervisors. The examples in this
+      manual use &kvm; running on &sle;.
+     </para>
+     <para>      
+      You must build at least two ISO images with configuration information:
+      one for the &admin_node; and one for the &worker_node;s. If you wish to
+      assign specific host names or customize individual &worker_node;s, then
+      you must create separate ISO images for each node containing the
+      customised files.
+     </para>
+     <para>
+      Additionally, you must make a separate copy of the downloaded VM disk
```

Comment:
try to avoid 'must': ...you need to make a separate...

Reply:
"Need to" is preferred? Oh! OK.

Reply:
Fixed.